turf/rhumb_bearing/_rhumb_bearing.py

```diff
@@ -0,0 +1,62 @@
+from math import fmod
+from typing import Dict, List, Union
+
+import numpy as np
+
+from turf.helpers import degrees_to_radians, radians_to_degrees
+from turf.helpers import Point
+from turf.invariant import get_coord
+
+
+def rhumb_bearing(
+    origin: Union[List, Point], destination: Union[List, Point], options: Dict = None
```

Comment:
The types here should be `Union[Sequence, Dict, Feature]`

Reply:
👍